translations/translation_de.xml

```diff
@@ -226,12 +226,12 @@
 		<text name="COURSEPLAY_HEADLAND_CORNER_TYPE_SMOOTH"  				text="weich" />
 		<text name="COURSEPLAY_HEADLAND_CORNER_TYPE_SHARP"  				text="scharf"/>
 		<text name="COURSEPLAY_HEADLAND_CORNER_TYPE_ROUND"  				text="rund" />
-		<text name="COURSEPLAY_CENTER_MODE"									text="Feldmitte (experimental)" />
+		<text name="COURSEPLAY_CENTER_MODE"									text="Feldmitte" />
 		<text name="COURSEPLAY_CENTER_MODE_UP_DOWN"							text="hoch/runter" />
 		<text name="COURSEPLAY_CENTER_MODE_CIRCULAR"						text="kreisförmig" />
 		<text name="COURSEPLAY_CENTER_MODE_SPIRAL"						    text="spiralförmig" />
-		<text name="COURSEPLAY_CENTER_MODE_LANDS"						    text="Lands" />
-		<text name="COURSEPLAY_NUMBER_OF_ROWS_PER_LAND"					    text="Number of rows per land" />
+		<text name="COURSEPLAY_CENTER_MODE_LANDS"						    text="Beete (innen nach aussen)" />
```

Comment:
Die Übersetzung Lands -> Beete finde ich nicht passende gewählt.
Es sind viel mehr Inseln die von innen nach außen abgearbeitet werden.
Bitte was besseres überlegen.
Wie gesagt vlt Inseln ?

Reply:
Inseln wäre ziemlich irreführend mit den Feldinseln die ja ebenfalls im Kursgeneratorfenster vorkommen.
In der realen Landwirtschaft spricht man bei dieser Art Arbeitsweise (vor Allem bei der Ernte oft das Standardmuster) tatsächlich eigentlich immer von "Beeten".  Das Feld wird gedanklich in eben diese unterteilt und diese dann in diesem Fall jeweils von innen nach außen abgearbeitet (deswegen das "innen nach außen" mit angeführt, weil es auch grundsätzlich auch noch andere Möglichkeiten gäbe).
Wurde mir so schon von mehreren Personen die selbst in dem Bereich tätig sind/waren unabhängig voneinander bestätigt. Und sogar in den Hochschul-Studiengängen wird konsequent dieser Begriff für genau diesen Zweck genutzt. Mit "Beete von innen nach außen abarbeiten" weiß in der Landwirtschaft vom Professor bis zum Lehrling eigentlich jeder auf Anhieb was gemeint ist.

Reply:
Naja die Frage ist ja nicht was ein gelernter Landwirt damit anfangen kann, sondern die User...

Reply:
Schon, aber nen passenderen Begriff seh ich da auch als Laie nicht für.